scripts/recreation_server/execute_recmodel_server.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import multiprocessing
+
+import natcap.invest.recreation.recmodel_server
+
+logging.basicConfig(format='%(asctime)s %(name)-20s %(levelname)-8s %(message)s', level=logging.DEBUG, datefmt='%m/%d/%Y %H:%M:%S ')
+
+args = {
+    'hostname': '10.240.0.4',
```

Comment:
A local IP definitely works since that's how we've been launching the Pyro server. Nevertheless I removed the address and added a comment since this is definitely a value that will change if we're ever using this script after setting up a new server.